### ISSUE TITLE
PHP basics setup: invalid make target

### ISSUE THREAD

**Issue by flipkickmedia** [grpc/grpc.io#664]
https://grpc.io/docs/languages/php/basics/#setup

You need grpc-php-plugin to help you generate proto files. You can build it from source:

$ cd grpc && git submodule update --init && make grpc_php_plugin

this fails...

```
$ make grpc_php_plugin
make: *** No rule to make target 'grpc_php_plugin'.  Stop.
```

**Comment by chalin** [comment]
Thanks for reporting this issue over the docs. A recent change to grpc Core seems to have invalidated the docs, as described in [grpc_php_plugin cannot be built as in quick start/tutorial for PHP · Issue #25350 · grpc/grpc](https://github.com/grpc/grpc/issues/25350), where @stanley-cheung wrote:

> We probably should update the docs. You can build it with bazel now:
> 
> ```
> ./tools/bazel build @com_google_protobuf//:protoc
> ./tools/bazel build src/compiler:grpc_php_plugin
> ```

Yes, let's update the docs 😄 

@stanley-cheung @gnossen @jtattermusch : Note that we had decided to use make-based builds (as opposted to using bazel) in the C++ quick start and basics tutorial. I think that we should do the same for the PHP quick start and tutorial.

**Comment by jtattermusch** [comment]
Starting from https://github.com/grpc/grpc/pull/24000, there is no way how to build the `grpc_*_plugin` with pure make at all.
One either needs to use bazel or cmake to build the grpc_php_plugin.

**Comment by chalin** [comment]
> One either needs to use bazel or cmake to build the grpc_php_plugin.

Got it. (Just to be clear, when I wrote "make-based builds" earlier, I meant cmake.)

> Both grpc.io quick start and https://github.com/grpc/grpc/issues/25350 should be updated accordingly.

I've subscribed to https://github.com/grpc/grpc/issues/25350 and so will see any updated that y'all post there. Note that the PHP quick start hasn't been updated in quite a while -- in particular, the C++ quick start updates made in the past 12 months (to document use of cmake) should be ported to the PHP quick start, or the PHP quick start should refer to the C++ quick start for the basic setup and gRPC build instructions.

Btw for next time: issues with the docs, like https://github.com/grpc/grpc/issues/25350, should probably be transferred to this repo.
